src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -4,21 +4,22 @@ public class RightTriangle extends Figure {
     private double side;
     private double height;
 
-    public RightTriangle(String color, double side) {
+    public RightTriangle(Color color, double side) {
         super(color);
         this.side = side;
         this.height = side * Math.sqrt(3) / 2;
     }
 
     @Override
     public void draw() {
-        System.out.println("Figure: triangle, area: " + getArea()
+        System.out.println("Figure: triangle, area: " + calculateArea()
                 + " sq.units, side: " + side + " units, height: "
                 + height + " units, color: " + getColor());
     }
 
     @Override
-    public double getArea() {
-        return 0.5 * side * height;
+    public double calculateArea() {
+        final double coefficient = 0.5;

```

Comment:
This is not a constant
Constant should be class level, static and final. Constant name should be in uppercase
For example private static final double FORMULA_COEFFICIENT = 0.5;